src/com/softwaret/kpdf/application/Application.kt

```diff
@@ -36,24 +44,48 @@ private fun Application.installFeatures() {
             configure(SerializationFeature.INDENT_OUTPUT, true)
         }
     }
+    install(Authentication) {
+        jwt {
+
+        }
+    }
     install(DefaultHeaders)
     install(PartialContent)
 }
 
-private fun Application.bindDI() {
+private fun setupDb() {
     Db.init()
+}
 
+@KtorExperimentalAPI
+private fun Application.bindDI() {
     di {
         bindControllers()
         bindInteractors()
-        bindServices()
+        bindServices(createServiceParameters())
     }
 }
 
 @KtorExperimentalLocationsAPI
 private fun Application.bindRouting() {
+
     routing {
         login(instance())
         register(instance())
     }
-}
\ No newline at end of file
+}
+
+private const val EXPIRATION_TIME: Long = 300_00
+
+@KtorExperimentalAPI
+private fun Application.createServiceParameters(): ServiceParameters {
```

Comment:
Wszystkie zależności zawierajace się w `application.conf` wyciągamy w pliku `Application.kt` i podajemy, aby potem opakowane w `data class` wstrzyknąć razem z DI

Reply:
Spoko, dodatkowo powinniśmy jakieś najbardziej czułe rzeczy wyłączyć z plików projektu tak żeby każdy musiał sobie setupować własny plik i z tego pliku gradle powinien odczytywać api key itd

https://stackoverflow.com/a/33136057/6592863

Reply:
@jstarczewski  Wywaliłem application.conf z trackowanych plików i dodałem go do gitignore, zobacz czy to jest k